integration-tests/contracts/ccipreader_test.go

```diff
@@ -94,11 +110,103 @@ func setupGetCommitGTETimestampTest(ctx context.Context, t *testing.T, finalityD
 		SimulatedBackend: sb,
 		Auth:             auth,
 		FinalityDepth:    finalityDepth,
+		UseHeavyDB:       useHeavyDB,
 	})
 
 	return s, finalityDepth, onRampAddress
 }
 
+func setupExecutedMessageRangesTest(ctx context.Context, t testing.TB, useHeavyDB bool) *testSetupData {
+	cfg := evmtypes.ChainReaderConfig{
```

Comment:
Do we need that step? Why don't you use the predefined config for the reader `DestReaderConfig` https://github.com/smartcontractkit/chainlink/blob/f986e0e321f6b7ec459b4f0edaa21d7c0e6f0f56/core/capabilities/ccip/configs/evm/contract_reader.go#L53

That way we also use the "proper" config

Reply:
Same applies to other chain reader inits